### DISCUSSION TITLE
how to swap alt and altgr?

### DISCUSSION THREAD

**Discussion by geschaenk** [sezanzeb/input-remapper#491] [Q&A]
I wonder if it's possible to swap left-alt (alt) and ISO Level3 Shift (alt gr)?
I'm using input-remapper 1.4.0 on ubuntu 22.04

**Comment by jonasBoss** [comment]
did you try `KEY_LEFTALT` and `KEY_RIGHTALT`?

**Reply by geschaenk** [reply]
Thank you that was helpful!
